components/brave_ads/browser/notification_helper_android.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_NOTIFICATION_HELPER_ANDROID_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_NOTIFICATION_HELPER_ANDROID_H_
+
+#include "base/macros.h"
+#include "base/memory/singleton.h"
+#include "base/memory/weak_ptr.h"
+#include "brave/components/brave_ads/browser/notification_helper.h"
+#include "chrome/browser/notifications/notification_channels_provider_android.h"
+
+
+namespace brave_ads {
+
+class NotificationHelperAndroid :
+    public NotificationHelper,
+    public base::SupportsWeakPtr<NotificationHelperAndroid> {
+ public:
+
+  static const int APP_NOTIFICATIONS_STATUS_UNDETERMINABLE = 0;
```

Comment:
Can these be included from elsewhere so we do not re-define the values?

Reply:
The function `Java_NotificationSystemStatusUtil_getAppNotificationStatus` returns `int`.
The constants are declared for readability instead of hard-coding well-known int values.

Reply:
The issue for me is that these values are changed in the future, and we would have no compiler warning whatsoever other than the app breaks

Reply:
These consts are private in Java and we care only about value returned by the function but not about consts.

Reply:
Ah apologies I did not know they were private